chat-app/src/App.css

```diff
@@ -0,0 +1,213 @@
+/* .App {
+  text-align: center;
+}
+
+.App-header {
+  background-color: #afb2b8;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: top;
+  font-size: calc(20px + 2vmin);
+  color: rgb(103, 81, 81);
+} */
+ * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+ }
+
+body {
+  background: #ebebeb;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.title {
+  margin: 20px 0;
+}
+
+.main {
+  border: 8px solid #dddddd;
+  border-radius: 20px;
+  overflow: hidden;
+  padding: 10px;
+}
+
+.name {
+  display: flex;
+  font-size: 32px;
+  font-weight: 700;
+  padding: 8px 16px;
+  color: #7e7e7e;
+  background: #ebebeb;
+}
+
+.room {
+  display: flex;
+  font-size: 32px;
+  font-weight: 700;
+  padding: 8px 16px;
+  color: #7e7e7e;
+  background: #ebebeb;
+}
+
+.room-button {
+  display: flex;
+  font-size: 24px;
+  font-weight: 700;
+  padding: 8px 16px;
+  margin: 0 12px;
+  /* color: blue; */
+  background-color: green;
+  color: #7e7e7e;
+  border: none;
+  border-radius: 25px;
+}
+
+.create-room-input {
+  border-radius: 5px;
+  padding: 8px 16px;
+  margin: 0 12px;
+}
+
+.create-room-button {
+  border-radius: 5px;
+  padding: 8px 16px;
+  /* margin: 0 12px; */
+}
+
+.name-input {
+  font-size: 24px;
+  font-weight: 700;
+  color: #7e7e7e;
+  flex-grow: 1;
+  border: none;
+  margin: 0px 12px;
+  outline: none;
+  background-color: #ebebeb;
+}
+
+.error-message {
+  padding: 8px 16px;
+  font-size: 24px;
+  font-weight: 700;
+  color: red;
+  font-weight: bold;
```

Comment:
The `font-weight` property is declared twice in the `.error-message` class. You can remove one of them to avoid redundancy.